src/main/kotlin/no/nav/familie/ef/sak/brev/FritekstBrevController.kt

```diff
@@ -10,13 +9,13 @@ import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 
 @RestController
-@RequestMapping(path = ["/api/manueltbrev"])
+@RequestMapping(path = ["/api/brev"])
 @ProtectedWithClaims(issuer = "azuread")
-class ManueltBrevController(private val brevClient: BrevClient) {
+class FritekstBrevController(private val frittståendeBrevService: FrittståendeBrevService) {
 
-    @PostMapping("")
-    fun lagManueltBrev(@RequestBody brevInnhold: JsonNode): Ressurs<ByteArray> {
-        return Ressurs.success(brevClient.lagManueltBrev(brevInnhold))
+    @PostMapping("/frittstående")
```

Comment:
Litt usikker på `å` i path her, burde kanskje bruke `a` som i `/vilkar`?

Reply:
å funker ikke i path, dessverre... Been there, done that :(

Reply:
Fikset i a422031